frontend/src/app/utils/srGraphQLErrorMessage.tsx

```diff
@@ -23,32 +23,54 @@ export function getHeader(message: string): string {
   return getErrorText(message, "header");
 }
 
+function createSupportMailToLink(msg: string, index: number) {
+  return (
+    <span key={index}>
+      {" "}
+      <a href="mailto:support@simplereport.gov">{msg}</a>{" "}
+    </span>
+  );
+}
+
+function createContactUsLink(msg: string, nextMsg: string, index: number) {
+  return (
+    <span key={index}>
+      {" "}
+      <a
+        target="_blank"
+        rel="noopener noreferrer"
+        href="https://www.simplereport.gov/contact-us"
+      >
+        {msg} {nextMsg}
+      </a>{" "}
+    </span>
+  );
+}
+
 export function getBody(message: string): JSX.Element {
   let errorText = getErrorText(message, "body");
   let errorTextArray = errorText.split(" ");
   let errorElementArray: any = [];
-  let addNewValue = true;
+  let addHTMLElement = true;
   errorTextArray.forEach((msg, index) => {
+    let nextMsg = errorTextArray[index + 1];
     if (msg.includes("support@simplereport.gov")) {
-      let addEndWhitespace = index !== errorTextArray.length - 1;
```

Comment:
haha I think I was overthinking things -- so I removed it! https://github.com/CDCgov/prime-simplereport/pull/5165#discussion_r1094809987 Thank you!